tests/test_environment.py

```diff
@@ -6,32 +6,31 @@
 from contextlib import contextmanager
 
 import pytest
-from twitter.common.contextutil import temporary_dir
 
-from pex import resolver
+from pex import resolver, vendor
```

Comment:
Is `vendor` intended to be a public API of PEX? If not, should it be prefixed with `_`?

Reply:
It is not, but this is a big problem right now with pex - there is no demarcation. I'll defer and keep the local style for now if you don't object. I had filed #584 a little while back to attack the problem.